src/app/rosetta/lib/block.ml

```diff
@@ -491,7 +491,7 @@ WITH RECURSIVE chain AS (
         FROM internal_commands i
         INNER JOIN blocks_internal_commands bic ON bic.internal_command_id = i.id
         INNER JOIN account_identifiers ai on ai.id = i.receiver_id
-        INNER JOIN accounts_created ac on ac.account_identifier_id = ai.id
+        LEFT JOIN accounts_created ac on ac.account_identifier_id = ai.id
```

Comment:
I think there should be no problem with the join becoming a `LEFT JOIN` since the only field from `accounts_created` we need is `creation_fee` and the OCaml type for it it was already `option int64`.

Reply:
Yes, that was my reasoning as well.